### ISSUE TITLE
FFMPEG --> extract video metadata

### ISSUE THREAD

**Issue by alagauche** [cablio/cs160#2]
I will use system calls in Python to extract video metadata and parse the results.

**Comment by alagauche** [comment]
Parsing the results is meh right now. This task is done as a bash script extract_video_metadata.sh in cs160/pipeline. I'm going to wait on getting the SSD to edit this code further. It's fairly trivial to run the FFMPEG commands, they are exactly as he has them on his website. The trick is going to be getting the web app to parse the results and then connecting it to the database, but that is a separate issue.
